src/libraries/Pool.sol

```diff
@@ -328,6 +332,10 @@ library Pool {
 
         bool exactInput = params.amountSpecified < 0;
 
+        if (!exactInput && (slot0Start.lpFee == LPFeeLibrary.MAX_LP_FEE)) {
```

Comment:
i wonder if it would make sense to just move this check to after `swapFee = ` and then check 
```
if (!exactInput && swapFee == LP...)
```

i know you fuzz that this cannot happen. But given it won't increase gas it feels like an extra safe way to do things.

Reply:
yeah i like that. increases gas by a little, but seems safer